### DISCUSSION TITLE
tutorials/authoring/01-getting-started

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#14] [course feedback]
# tutorials/authoring/01-getting-started

https://www.apollographql.com/tutorials/authoring/01-getting-started

**Comment by R4ph-t** [comment]
test

**Comment by R4ph-t** [comment]
test test

**Comment by R4ph-t** [comment]
another test for homer-giscus!

**Reply by mabuyo** [reply]
test to reply

**Comment by R4ph-t** [comment]
test no link
